### PR TITLE
Remove scipy upper bound for streaming extractors

### PR DESCRIPTION
**Pull request by h-mayorquin**
This was solved:
https://github.com/int-brain-lab/ibllib/issues/753#event-13133745384

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -65,7 +65,7 @@ extractors = [
     "pyedflib>=0.1.30",
     "sonpy;python_version<'3.10'",
     "lxml", # lxml for neuroscope
-    "scipy<1.13",
+    "scipy",
     "ONE-api>=2.7.0", # alf sorter and streaming IBL
     "ibllib>=2.32.5", # streaming IBL
```
Comment (alejoe91) [inline pyproject.toml:70]: this should be updated as well no?

Comment (h-mayorquin) [reply pyproject.toml:70]: Yes!

Comment (h-mayorquin) [inline pyproject.toml:70]:
```suggestion
    "ibllib>=2.36.0", # streaming IBL
```

Comment (h-mayorquin) [reply pyproject.toml:70]: Ah, strange marking.

**Review by alejoe91 — APPROVED** [review]

**Comment by samuelgarcia** [comment]
merci

**Pull request merged**